kubernetes/operationcode_python_backend/base/deployment.yaml

```diff
@@ -102,9 +102,15 @@ spec:
         - name: EXTRA_HOSTS
           value: api.staging.operationcode.org
         - name: RECAPTCHA_PUBLIC_KEY
-          value: override_me
+          valueFrom:
```

Comment:
This is being done since kustomize doesn't like you to have a `value` in the base, and `valueFrom` in overlay.